src/test/java/seedu/address/logic/commands/GetEditRecipeCommandTest.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.showEditRecipeCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showRecipeAtIndex;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_RECIPE;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_RECIPE;
+import static seedu.address.testutil.TypicalRecipes.getTypicalWishfulShrinking;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.recipe.Recipe;
+
+public class GetEditRecipeCommandTest {
+    private Model model = new ModelManager(getTypicalWishfulShrinking(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        Recipe recipeToEdit =
+                model.getFilteredRecipeList().get(INDEX_FIRST_RECIPE.getZeroBased());
+        GetEditRecipeCommand getEditRecipeCommand = new GetEditRecipeCommand(INDEX_FIRST_RECIPE);
+
+        String expectedMessage = String.format(GetEditRecipeCommand.MESSAGE_GET_EDIT_RECIPE_SUCCESS,
+                recipeToEdit.toString());
+
+        ModelManager expectedModel = new ModelManager(model.getWishfulShrinking(), new UserPrefs());
+
+        showEditRecipeCommandSuccess(getEditRecipeCommand, model, expectedMessage, recipeToEdit,
+                expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexUnfilteredList_throwsCommandException() {
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredRecipeList().size() + 1);
+        GetEditRecipeCommand getEditRecipeCommand = new GetEditRecipeCommand(outOfBoundIndex);
+
+        assertCommandFailure(getEditRecipeCommand, model, Messages.MESSAGE_INVALID_RECIPE_DISPLAYED_INDEX);
+    }
+
+    //@Test
```

Comment:
Same as previous tests

Reply:
Same I commented it out because not sure if its relevant. It looks like its testing the model and this command does not modify model instead it modifies CommandResult which has been tested.